src/components/invite-dialog/index.tsx

```diff
@@ -22,6 +22,7 @@ export interface Properties {
   maxUses: number;
   inviteUrl: string;
   assetsPath: string;
+  isUserInFullScreenModeAndInWorlds: boolean;
```

Comment:
I don't think checking for full screen is necessary. The invite dialog doesn't care if you're in full screen mode or not. The message only matters if you're in worlds or not.

Reply:
okay, updating